gravitee-apim-e2e/api-test/apis/gateway-instances.spec.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { beforeAll, describe, expect, test } from '@jest/globals';
+import { GatewayApi } from '@management-apis/GatewayApi';
+import { forManagementAsAdminUser, forManagementAsApiUser } from '@client-conf/*';
+import { fail } from '../../lib/jest-utils';
+
+const orgId = 'DEFAULT';
+const envId = 'DEFAULT';
+const gateway_admin = new GatewayApi(forManagementAsAdminUser());
+const gateway_user = new GatewayApi(forManagementAsApiUser());
+
+describe('Gateway tests', () => {
```

Comment:
`Gateway tests` is a bit misleading as we are not testing the Gateway directly but the /gateway endpoint of the management API
```suggestion
describe('GatewayApi', () => {
```